javascript-algorithms-and-data-structures/learn-form-validation-by-building-a-calorie-counter/styles.css

```diff
@@ -1,93 +1,94 @@
+:root {
+  --light-text: #f5f6f7;
+  --dark-text: #0a0a23;
+  --fcc-blue: #1b1b32;
+  --light-yellow: #fecc4c;
+  --dark-yellow: #feac32;
+  --light-pink: #ffadad;
+  --dark-red: #850000;
+  --light-green: #acd157;
+}
+
 body {
-  background: gray;
-  font-family: 'Segoe UI', Arial;
+  font-family: "Lato", Helvetica, Arial, sans-serif;
+  font-size: 18px;
+  background-color: var(--fcc-blue);
+  color: var(--light-text);
+}
+
+h1 {
+  text-align: center;
 }
 
 .container {
-  margin: 30px auto;
-  background: white;
-  padding: 30px 40px;
-  border-radius: 5px;
-  width: 400px;
+  width: 90%;
+  max-width: 680px;
 }
 
-.center {
-  text-align: center;
+h1,
+.container,
+.output {
+  margin: 20px auto;
 }
 
-h4,
-h3 {
-  margin: 5px;
+label,
+legend {
+  font-weight: bold;
 }
 
-.grid {
-  display: grid;
-  grid-template-columns: 120px 300px;
-  grid-row-gap: 5px;
-  align-items: center;
+.input-container {
+  display: flex;
+  flex-direction: column;
 }
 
 button {
-  border-radius: 4px;
-  background-color: white;
-  font-family: 'Segoe UI', Arial;
-  font-size: 16px;
-  margin: 10px auto;
-  border: none;
-  padding: 5px 10px;
+  outline: none;
   cursor: pointer;
+  text-decoration: none;
+  background-color: var(--light-yellow);
+  border: 2px solid var(--dark-yellow);
 }
 
-button:focus {
-  outline: 0;
+.clear {
+  background-color: var(--light-pink);
+  color: var(--dark-red);
+  border-color: var(--dark-red);
 }
 
-.btn-solid {
-  background-color: rgb(40, 167, 69);
-  color: white;
-  border: 1px solid rgb(40, 167, 69);
+button,
+input,
+select {
+  min-height: 24px;
+  color: var(--dark-text);
 }
 
-.btn-outline {
-  color: rgb(40, 167, 69);
-  border: 1px solid rgb(40, 167, 69);
+fieldset,
+label,
+button,
+input,
+select {
+  margin-bottom: 10px;
 }
```

Comment:
I think we should use a class here instead

Reply:
I played around with the code and added some classes to these elements, but there doesn't seem to be a huge benefit.

We could add class names like `budget-input`, `breakfast`, `lunch`, `dinner`, `exercise`, `entry-name-label`, `entry-name-input`, and so on. However, this could add a few more steps and make the code a bit harder to read. We would also need to modify some of the template strings in the JS to include specific class names.

We could use a utility class instead like `spacing` instead and add that to all the elements, though again it could add more steps and require updating some JS template strings.

Since the page is pretty simple and we want to add a bottom margin to all of these elements uniformly, I think it's easier to stick with type selectors here.